uiowa/uiowa01
# Feature: Collegiate Theme HDS Fixes

Initial work on #198, #210, and #218.

# How to Test

#198 

1. Run si collegiate_basic
2. Go to Appearance and check to see if new screenshot is displaying
3. Go to Appearance > Theme settings and check capitalization on "Nav"

#210 

1. Run si collegiate_basic
2. Create a basic page
3. Edit a layout and create a card with and without a link

#218 

1. Run si collegiate_basic
2. Create a basic page
3. Edit a layout and create a card, banner, feature block, noteworthy, and statistic with and without title text
